debian/endless-ca-cert.postinst

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+CONF_FILE=/etc/ca-certificates.conf
+CONF_FILE_NEW="$CONF_FILE".dpkg-new
+CONF_FILE_OLD="$CONF_FILE".dpkg-old
+CA_CONF=endless/endless-ca.crt
+
+if [ "$1" = configure ]; then
+    # Add the CA cert to the configuration file if there's no existing
+    # declaration
+    if [ -f "$CONF_FILE" ]; then
+        cp "$CONF_FILE" "$CONF_FILE_NEW"
+        if ! grep -q "^!\?${CA_CONF}$" "$CONF_FILE_NEW"; then
+            echo "$CA_CONF" >> "$CONF_FILE_NEW"
+        fi
+        if cmp -s "$CONF_FILE" "$CONF_FILE_NEW"; then
+            rm -f "$CONF_FILE_NEW"
+        else
+            mv -f "$CONF_FILE" "$CONF_FILE_OLD"
+            mv "$CONF_FILE_NEW" "$CONF_FILE"
+        fi
```

Comment:
This seems a touch overly complicated. Can we check if the file has the $CA_CONF and if it does just exit and otherwise do the backup/copy/append/replace?

Reply:
I copied that from `ca-certificates`, but I agree it's unnecessary to do the copy if there's no need.